example/simple/src/main.c

```diff
@@ -91,7 +91,7 @@ static void simple_complete(ocf_cache_t cache, void *priv, int error)
  */
 int initialize_cache(ocf_ctx_t ctx, ocf_cache_t *cache)
 {
-	struct ocf_mngt_cache_config cache_cfg = { .name = "cache1" };
+	struct ocf_mngt_cache_config cache_cfg = { .name = "cache1", .name_len = env_strnlen(cache_cfg.name, OCF_CACHE_NAME_SIZE) };
```

Comment:
name_len should be assigned "env_strnlen(...) + 1".   Or even better" sizeof("cache1")

Reply:
changed